src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="calendar calendar--start-day--sun calendar--month-length--28 calendar--month-length--29 calendar--month-length--30 calendar--month-length--31">
```

Comment:
```suggestion
    <div class="calendar calendar--start-day--sun calendar--month-length--31">
```
There should be just one modifier and a user should be able to change it and see the rerendered layout of the calendar